### ISSUE TITLE
Compatibility with easy Redmine?

### ISSUE THREAD

**Issue by fcappz** [raafael911/redmine_once_assigned_ever_watcher#1]
We're considering getting EasyRedmine for functionality not available via other Redmine plugins. Has this plugin been tested or used with EasyRedmine?  Thanks.

**Comment by raafael911** [comment]
No, it hasn't been tested with EasyRedmine. It's because I don't know what EasyRedmine is. If you post a link here I can see and test it.

**Comment by fcappz** [comment]
Thanks for your reply. I'm not sure if you will be able to get access to EasyRedmine. Hee is some information: http://www.redmine.org/plugins/easyredmine and their website is here: http://www.easyredmine.com .  Thanks in advance for any help.

**Comment by raafael911** [comment]
Okay you are right, I have to register for a free trial and I do not want to do that. But they say that EasyRedmine is just a collection of extensions for Redmine. Therefore I would assume that the plugin will work. You can simply try it because this plugin doesn't make any irreversible changes to redmine. If it doesn't work just delete the plugin folder and restart the webservice.
